src/Init/Core.lean

```diff
@@ -1908,6 +1908,13 @@ instance Pi.instSubsingleton {α : Sort u} {β : α → Sort v} [∀ a, Subsingl
 
 /-! # Squash -/
 
+theorem true_equivalence (α : Sort u) : Equivalence fun _ _ : α ↦ True :=
```

Comment:
```suggestion
theorem equivalence_true (α : Sort u) : Equivalence fun _ _ : α ↦ True :=
```

Reply:
These new names are better, and should avoid Mathlib breaking :)